pkgs/by-name/lo/loopwm/update.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename "$ROOT")" == "loopwm" || ! -f "$ROOT/package.nix" ]]; then
+    echo "error: Not in the loopwm folder" >&2
+    exit 1
+fi
+
+PACKAGE_NIX="$ROOT/package.nix"
+
+# Grab latest release version
+LOOPWM_LATEST_VER="$(curl --fail -s ${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} "https://api.github.com/repos/MrKai77/Loop/releases/latest" | jq -r '.tag_name' | sed 's/^v//')"
+LOOPWM_CURRENT_VER="$(grep -oP 'version = "\K[^"]+' "$PACKAGE_NIX")"
+
+if [[ "$LOOPWM_LATEST_VER" == "$LOOPWM_CURRENT_VER" ]]; then
+    echo "loopwm is up-to-date"
+    exit 0
+fi
+
+get_hash() {
+    # $1: URL
+    nix-hash --to-sri --type sha256 "$(nix-prefetch-url --type sha256 "$1")"
+}
+
+replace_hash_in_file() {
+    # $1: file
+    # $2: new hash
+    sed -i "s#hash = \".*\"#hash = \"$2\"#g" "$1"
+}
+
+replace_version_in_file() {
+    # $1: file
+    # $2: new version
+    sed -i "s#version = \".*\";#version = \"$2\";#g" "$1"
+}
+
+LOOPWM_DARWIN_HASH="$(get_hash "https://github.com/MrKai77/Loop/releases/download/${LOOPWM_LATEST_VER}/Loop.zip")"
+
+replace_hash_in_file "$PACKAGE_NIX" "$LOOPWM_DARWIN_HASH"
```

Comment:
We don't need to complicate things, we can just inline this and short the script with it